src/utils/processSystem.cpp

```
@@ -1,32 +1,42 @@
+uint ProcessSystem::nowSuccessCount = 0;
+uint ProcessSystem::failCount = 0;
+uint ProcessSystem::maxFailCount;
+uint ProcessSystem::maxTaskCount;
+bool ProcessSystem::isCanRun = false;
+vector<thread> ProcessSystem::threads = vector<thread>();
+
 
-namespace fs = std::filesystem;
 ProcessSystem::ProcessSystem()
 {
+    threadNumber = 1;
```

cppcheck

[performance] Variable 'threadNumber' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)